core/templates/dev/head/pages/exploration_editor/simple_editor_tab/questions/MultipleChoiceInputCheckerServiceSpec.js

```diff
@@ -18,19 +18,82 @@
 
  describe('Multiple Choice Input Checker Service', function() {
```

Comment:
Leading white space here and also before each line. I am not sure , but It may create linting issue. I request @andromfins to confirm?